bundle/config/mutator/resourcemutator/process_static_resources.go

```diff
@@ -32,6 +33,27 @@ func (p processStaticResources) Apply(ctx context.Context, b *bundle.Bundle) dia
 		return diag.FromErr(err)
 	}
 
+	// only YAML resources need to have paths normalized, before normalizing paths
+	// we need to resolve variables because they can change path values:
+	// - variable can be used a prefix
+	// - path can be part of a complex variable value
+	diags := bundle.ApplySeq(
+		ctx,
+		b,
+		// Reads (dynamic): * (strings) (searches for variable references in string values)
+		// Updates (dynamic): resources.* (strings) (resolves variable references to their actual values)
+		// Resolves variable references in 'resources' using bundle, workspace, and variables prefixes
+		mutator.ResolveVariableReferencesOnlyResources(
```

Comment:
Could you please also add coverage in an acceptance test that variable resolution happens before glob expansion.